### DISCUSSION TITLE
Image Masks

### DISCUSSION THREAD

**Discussion by mkewal** [CancerDataScience/CrowdsourcingDataset-Amgadetal2019#12] [Q&A]
Should the image masks be coming up as black images, or is there an error when I am downloading the masks?

**Comment by kheffah** [comment]
@mkewal Sorry for the delay in getting back to you. The masks just look black then you view them using, say, the native Windows photo viewer. This is because this is a multi-class classification problem and the pixels have specific meaning. So a value of 1 means _mostly_tumor_, for example. Use something like this to view the masks instead:
```python
import matplotlib.pylab as plt
from imageio import imread

mask = imread('/path/to/mask.png')
plt.imshow(mask)
plt.show()
```
